client/components/catalog/SelectOrder.vue

```diff
@@ -6,7 +6,7 @@
           <template v-slot:activator="{ on: tooltip }">
             <v-btn
               v-on="{ ...menu, ...tooltip }"
-              icon text
+              icon
```

Comment:
Color is defined on icon rather than on btn, if you move it here it should solve hover as well ;)